### ISSUE TITLE
Invisible wall-smashing but otherwise harmless creature found on asteroid that I'm almost certain wasn't adminbus

### ISSUE THREAD

**Issue by Exxion** [d3athrow/vgstation13#7608]
I did ahelp and ask if it was adminbus, but nobody responded. I don't see anything in the log about spawning it, either. And I don't think anyone VVed it, either, so, sadly, I can't be more specific. If you want to check the logs, I first "saw" the thing at roughly 16:13:22 (timestamp on the log) today, Thursday, January 7, though due to its properties it could have been (and likely was) present and following me long before that.
I was wandering on the mining asteroid, heading to the vox station from the mining station, when I saw a bunch of drips of blood and decided to follow them into a little cave that had been mined into. Inside, I looked around a bit. The only thing I saw before getting scared and running away was a dead basilisk.
When I got to the vox station, walls near me started breaking, with the message "The animal smashes the plating!" or something like that. The name was definitely just animal. This kept happening until I left the trading post. Also, the animal was dense, as I found that occasionally I could not move in a certain direction.
One weird thing was that I put my jetpack into a nitrogen tank to refill it, but the tank's menu stopped responding after a few seconds and when I closed it I couldn't reopen it, but that kind of thing just happens sometimes and may not be related.
It did not damage me or anything, only broke walls and blocked my path occasionally.

TL;DR: Probably a runtime. Are there runtime logs?

**Comment by Kurfursten** [comment]
http://ss13.undo.it/logs/Dev/runtime/

Spooky stuff, but no "animal" appears in runtimes

**Comment by Zth--** [comment]
We need more of these spooky things

**Comment by Exxion** [comment]
I can confirm it was spooky as fuck and I can also confirm I have no earthly idea what the fuck happened if nothing shows up in any logs

**Comment by Exxion** [comment]
OK I think I figured out _what_ it is I encountered, but not how the fuck it got there
All the information seems to point to it being a base /mob/living/simple_animal/hostile/asteroid
It deals 0 damage, is named animal, has environment_smash = 2, and has no icon_state, if I'm not mistaken
But _how the fuck did it get there_?

**Comment by Kurfursten** [comment]
Perhaps a bug in the asteroid monster spawning code

**Comment by Exxion** [comment]
I don't see anything obvious, but I'm rather tired and may be missing something
The only thing I see is that there is, for some reason, a slash at the end of its type definition
Which really seems like it shouldn't do anything but who knows because BYOND a shit

**Comment by Kurfursten** [comment]
That would generally have the opposite effect in any case

**Comment by PJB3005** [comment]
Goddamnit we need [IT'S A FEATUUUUURE] back

**Comment by CptWad** [comment]
You know, I have had weird issues with random spots being dense. For whatever reason it only happens when I am a mime. Sometimes a spot  near me gets dense. I cant walk through it and I half to walk around. Pretty weird since I know its not my wall spell (as it will happen next to me, not on top of). It goes away after a while and I always thought maybe it was an admin who was dense following me around or something.
I guess it warrants me looking into

**Comment by Exxion** [comment]
HAHA I FIGURED IT OUT JESUS FUCK

It's the traitor Lazarus capsules. No checks against spawning /mob/living/hostile/simple_animal/asteroid and the blood I mentioned in the OP that I followed into the cave was from two people killed by the only person that round (or day, for that matter) to buy the Master Trainer's Belt.

I'm only 99.9% sure because for some fucking reason Lazarus capsule throws aren't logged unless they hit a mob.
